starter-code/src/clue.js

```diff
@@ -85,10 +126,61 @@ name: Patio
 */
 
 // Characters Collection
-var charactersArray = [];
-
-// Rooms' Collection
-var roomsArray = [];
+var charactersArray = [
+  mrGreen,
+  drOrchid,
+  profPlum,
+  missScarlet,
+  mrsPeacock,
+  mrMustard
+];
 
 // Weapons Collection
-var weaponsArray = [];
+var weaponsArray = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+];
+
+// Rooms Collection
+var roomsArray = [
+  "dinningRoom",
+  "conservatory",
+  "kitchen",
+  "study",
+  "library",
+  "billardRoom",
+  "lounge",
+  "ballroom",
+  "hall",
+  "spa",
+  "livingRoom",
+  "obseratory",
+  "theatre",
+  "guestHouse",
+  "patio"
+];
+
+//Random Selector
+function randomSelector(array) {
+  var random = array[Math.floor(Math.random() * array.length)];
+  return random;
+}
+
+function pickMistery(charactersArray, weaponsArray, roomsArray) {
```

Comment:
here, you don't need to pass anything to the function as a parameter. If you remove them, you pass four more tests 

```
function pickMistery() {
  var christina = [];

  var char = randomSelector(charactersArray);
  var roo = randomSelector(roomsArray);
  var weap = randomSelector(weaponsArray);

  christina.push(char, roo, weap);

  return christina;
}
```

This is because the randomSelector() already takes the array as an parameter and you call this inside this function.